pubspec.yaml

```diff
@@ -8,12 +8,12 @@ documentation: https://docs.revenuecat.com/docs
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
-  flutter: ">=1.12.13+hotfix.6 <2.0.0"
```

Comment:
* The Flutter constraint should not have an upper bound.
  In your pubspec.yaml the constraint is currently `>=1.12.13+hotfix.6 <2.0.0`.
  
  You can replace that with just the lower bound: `>=1.12.13+hotfix.6`.
  
  See https://dart.dev/go/flutter-upper-bound-deprecation